Common subdirectories: a/doc and b/doc
Common subdirectories: a/examples and b/examples
Common subdirectories: a/hooks and b/hooks
diff -u a/Makefile b/Makefile
--- a/Makefile	2016-12-13 17:40:45.080512579 +0000
+++ b/Makefile	2016-12-13 17:41:10.955513196 +0000
@@ -1,7 +1,7 @@
 
 $(phony all): manfiles taskopen.pl
 
-taskopen.pl: clean manfiles
+taskopen.pl: manfiles
 	sed s',#PATH_EXT=.*,&\nPATH_EXT=$(PREFIX)/share/taskopen/scripts,' taskopen > taskopen.pl
 
 manfiles: 
Common subdirectories: a/scripts and b/scripts
